src/main/java/frc/robot/subsystems/claw/ClawStateMachine.java

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems.claw;
+
+import frc.robot.lib.statemachine.StateMachine;
+
+/** Add your docs here. */
+public class ClawStateMachine extends StateMachine {
+    
+    public static OpenState openState = new OpenState();
+    public static ClosedCone closedCone = new ClosedCone();
+    public static ClosedCube closedCube = new ClosedCube();
+
+    public ClawStateMachine() {
+        openState.build();
+        closedCone.build();
+        closedCube.build();
+
+        setCurrentState(openState);
```

Comment:
Just a note here, we might not want to start in this state. When we start a match, we will ideally have some kind of pre-load. Not sure how we should handle that. This is fine for now.